### PR TITLE
Add third-party expected GTK color names

### PR DESCRIPTION
**Pull request by lainsce**
These are seen in other apps than elementary's; adapted some to be different for dark style too.

### CODE REVIEW

File: src/widgets/_index.scss [hunk 1]
```diff
@@ -25,6 +25,23 @@ messagedialog,
 @define-color theme_selected_fg_color #{"" + $fg-color};
 @define-color wm_shadow #{rgba(black, 0.2)};
 
+// Things other apps expect
+@define-color theme_text_color #{'@text_color'};
+@define-color theme_bg_color #{'@bg_color'};
+@define-color theme_base_color #{'@base_color'};
```
Comment (danrabbit) [inline src/widgets/_index.scss:31]: I don't think this is defined yet

File: src/widgets/_index.scss [hunk 2]
```diff
@@ -25,6 +25,23 @@ messagedialog,
...
+@define-color selected_bg_color #{'alpha(@accent_color_100, 0.3)'};
+@define-color selected_fg_color #{'@accent_color_900'};
```
Comment (danrabbit) [inline src/widgets/_index.scss:33]: Probably should update the `selection` element to use these values so they always match

File: src/widgets/_index.scss [hunk 3]
```diff
@@ -25,6 +25,23 @@ messagedialog,
...
+@define-color theme_selected_bg_color #{'@selected_bg_color'};
+@define-color insensitive_fg_color #{'@insensitive_color'};
+
+@if $color-scheme == "dark" {
+    @define-color selected_bg_color #{'alpha(@accent_color_700, 0.3)'};
+    @define-color selected_fg_color #{'@accent_color_100'};
+    @define-color theme_selected_bg_color #{'@selected_bg_color'};
+    @define-color theme_selected_fg_color #{'@selected_fg_color'};
```
Comment (danrabbit) [inline src/widgets/_index.scss:41]: If these are already defined this way normally, then this isn't necessary to define twice

Comment (lainsce) [reply src/widgets/_index.scss:41]: It's how it was on the old stylesheet, should I just delete them?

File: src/widgets/_index.scss (hunk 2 above)
Comment (lainsce) [reply src/widgets/_index.scss:33]: Will do.

File: src/widgets/_index.scss (hunk 1 above)
Comment (lainsce) [reply src/widgets/_index.scss:31]: I'll define it as `bg-color(1)` to match previous styling on old stylesheet.

**Pull request closed without merging**